pkg/server/registration_test.go

```diff
@@ -0,0 +1,238 @@
+package server
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/spiffe/spire/proto/api/registration"
+	"github.com/spiffe/spire/proto/common"
+	"github.com/spiffe/spire/proto/server/datastore"
+	"github.com/spiffe/spire/test/mock/proto/server/datastore"
+	"github.com/spiffe/spire/test/mock/server/catalog"
+	"github.com/stretchr/testify/suite"
+)
+
+type RegistrationServerTestSuite struct {
+	suite.Suite
+	t                  *testing.T
+	ctrl               *gomock.Controller
+	registrationServer *registrationServer
+	logHook            *test.Hook
```

Comment:
do we need logHook. I did not see this being used anywhere..

Reply:
Good catch, I will remove it.